docs/application/native/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,634 @@
+        <ui-application appid="org.tizen.sample" exec="@BINDIR@/sample" nodisplay="false" multiple="false" type="capp" taskmanage="true" launch_mode="caller">
+            <label>Sample</label>
+            <icon>@DESKTOP_ICON@</icon>
+        </ui-application>
+        <privileges>
+            <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+        </privileges>
+    </manifest>
+    ```
+
+2.  Send launch request:
+
+    You can send launch request using `app_control_h`. `app_control_h` contains the component-based application's ID and the component ID. The component ID is optional. If you do not set the component ID, the main component instance will be created.
+
+    You can launch a component-based application as shown in the following code:
+    ```
+    static int __app_control_send(const char *app_id, const char *component_id,
+            void *user_data)
+    {
+        app_control_h handle = NULL;
+        int ret;
+
+        ret = app_control_create(&handle);
+        if (ret != APP_CONTROL_ERROR_NONE)
+            return ret;
+
+        ret = app_control_set_app_id(handle, app_id);
+        if (ret != APP_CONTROL_ERROR_NONE)
+            return ret;
+
+        if (component_id) {
+            ret = app_control_set_component_id(handle, component_id);
+            if (ret != APP_CONTROL_ERROR_NONE) {
+                app_control_destroy(handle);
+                return ret;
+            }
+        }
+
+        ret = app_control_set_launch_mode(handle,
+                APP_CONTROL_LAUNCH_MODE_GROUP);
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+
+        ret = app_control_send_launch_request_async(handle,
+                __app_control_result_cb, NULL, user_data);
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+
+        return 0;
+    }
+
+    static void __button_clicked_cb(void *data, Evas_Object *obj,
+        void *event_info) {
+        __app_control_send("org.tizen.base-component", "base-frame", NULL);
+    }
+    ```
+
+<a name="group_launch"></a>
+## Group Launch Management
+
+The component-based application also provides an [application group feature](../app-management/app-controls.md#group). To use the application group feature, follow these steps:
+
+1. Declare the launch privilege in the manifest file:
+    ```
+    <component-based-application appid="org.tizen.base-component" exec="@BINDIR@/base-component" nodisplay="false" multiple="false" type="capp">
+        <frame-component id="base-frame" launch_mode="caller" main="true" icon-display="false" taskmanage="true">
+          <icon>org.tizen.sample.png</icon>
+          <label>FrameComponent</label>
+          <label xml:lang="en-us">FrameComponent</label>
+          <label xml:lang="ko-kr">FrameComponent[KOR]</label>
+        </frame-component>FrameComponent</label>
```

Comment:
Just for check : 

`</frame-component>FrameComponent</label>`

Isn't
```
</frame-component>
<label>FrameComponent</label>
```
 correct?

Also please check the indentations in the code blocks, too.